drivers/npmx/npmx_shell.c

```diff
@@ -4183,8 +4327,8 @@ SHELL_STATIC_SUBCMD_SET_CREATE(sub_ldsw_mode,
 
 /* Creating dictionary subcommands (level 3 command) array for command "sub_ldsw_ldo_voltage". */
 SHELL_STATIC_SUBCMD_SET_CREATE(sub_ldsw_ldo_voltage,
-			       SHELL_CMD(set, NULL, "Set LDO voltage", cmd_ldsw_ldo_voltage_set),
 			       SHELL_CMD(get, NULL, "Get LDO voltage", cmd_ldsw_ldo_voltage_get),
+			       SHELL_CMD(set, NULL, "Set LDO voltage", cmd_ldsw_ldo_voltage_set),
```

Comment:
I would actually leave it as it is. It is a refactoring change and should be done in https://github.com/NordicSemiconductor/npmx-zephyr/pull/73#pullrequestreview-1660204831 . Moreover, @wilu-nordic claims that setters should be placed before getters, to be consistent with nrfx and npmx Coding standard.